src/behavior-property-observer.js

```diff
@@ -36,11 +60,12 @@ export class BehaviorPropertyObserver {
   * @param newValue The new value to set.
   */
   setValue(newValue: any): void {
-    let oldValue = this.currentValue;
+    const oldValue = this.currentValue;
+    const coercedValue = this.coerce !== undefined ? this.coerce(newValue) : newValue;
```

Comment:
nit pick- positive case first please.  `this.coerce === undefined ? newValue : ....`

Reply:
👍